src/uiSelectSingleDirective.js

```diff
@@ -51,6 +51,15 @@ uis.directive('uiSelectSingle', ['$timeout','$compile', function($timeout, $comp
 
       scope.$on('uis:select', function (event, item) {
         $select.selected = item;
+        var locals = {};        
+        locals[$select.parserResult.itemName] = item;
+
+        $timeout(function(){
+          $select.onSelectCallback(scope, {
```

Comment:
I did some checking with the unit test if this is the case but then it will fail. Why do you think it should be $select.scope?